docs/Game/Unreal/getting-started/04-mint-nft.md

```diff
@@ -27,7 +27,7 @@ The response is a 'ticket'.
 The session saved during Init will be used to open metamask. Metamask will show a popup to sign or confirm the transaction for that ticket.
```

Comment:
MetaMask?  You're writing it like this a lot.

And here you wrote "metamask" and Metamask".

Reply:
Done